cmd/fleetctl/debug.go

```diff
@@ -66,7 +66,8 @@ func writeFile(filename string, bytes []byte, mode os.FileMode) error {
 }
 
 func outfileName(name string) string {
-	return fmt.Sprintf("fleet-%s-%s", name, nowFn().Format(time.RFC3339))
+	time := strings.Replace(nowFn().Format(time.RFC3339), ":", "-", -1)
```

Comment:
Instead of using `strings.Replace`, you should just use a custom format string. For example `Format("20060102150405Z")`. Also, since we use `-` for separating the timestamp from the rest of the archive name, it might be nice to just omit delimiters entirely.

Reply:
I was torn between using a custom format or sticking with a well-known RFC, but absolutely it makes sense, and the timestamp without dashes is way more clear. Thank you!